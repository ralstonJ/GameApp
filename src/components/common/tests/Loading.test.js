import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Loading from '../../common/Loading';

afterEach(cleanup);

describe('Loading Component', () => {
  it('matches the snapshot', () => {
    const { asFragment } = render(<Loading />);
    expect(asFragment()).toMatchSnapshot();
  });
});
