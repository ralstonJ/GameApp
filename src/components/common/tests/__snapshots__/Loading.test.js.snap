// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Component matches the snapshot 1`] = `
<DocumentFragment>
  <div>
     Loading... 
  </div>
</DocumentFragment>
`;
